containerbuddy/config.json

```diff
@@ -1,11 +1,12 @@
 {
-  "consul": "consul:8500",
+  "consul": "{{.CONSUL}}:8500",
+  "onStart": "/opt/containerbuddy/onstart.sh",
   "services": [
     {
-      "name": "wordpress",
+      "name": "app",
```

Comment:
the default config on the pre-built nginx container is looking for "app" in consul, I did not see an obvious way of changing this since it's already baked into the image.